guides/common/modules/proc_upgrading-packages-on-a-host.adoc

```diff
@@ -0,0 +1,15 @@
+[id="upgrading-packages-on-a-host_{context}"]
+= Upgrading Packages on a Host
+
+The package upgrade process takes place via Katello agent or remote execution, depending on your configuration.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
+. Click the name of the host you want to modify.
+. Go to *Content* tab, then click the *Packages* tab.
+. Select the *Upgradable* filter from the *Status* box to list the upgradable packages.
+. Click *Upgrade*.
+
+If a package has more than one available upgrade version, only the latest upgradable version is displayed.
+
+You can run a remote execution job immediately, or select to customize the remote execution job.
```

Comment:
What can we add further? Because remote execution is explained separately at another place. And, it has mainly two ways (SSH, Ansible) in customizable jobs for which end-users are well aware during this process.

Reply:
Perhaps not elaborate on REX, but on the upgrade options.
When the user clicks the dropdown menu of the **Upgrade** button, there are two options:

- **Upgrade via remote execution** -- to run remote execution immeditaly
- **Upgrade via customized remote execution** -- to customize the REX job

If the user just clicks the **Upgrade** button, the first option is what happens.

Reply:
We can document about `Katello Agent` as well. Or remove the same from the introductory paragraph.

Reply:
> Perhaps elaborate not on REX, but on the upgrade options. When the user clicks the dropdown menu of the **Upgrade** button, there are two options:
> 
>     * **Upgrade via remote execution** -- to run the REX job immediately
> 
>     * **Upgrade via customized remote execution** -- to customize the REX job
> 
> 
> If the user just clicks the **Upgrade** button, the first option is what happens.

Covered this :+1: